scm/driver/stash/git.go

```diff
@@ -20,7 +20,14 @@ type gitService struct {
 }
 
 func (s *gitService) CreateBranch(ctx context.Context, repo string, params *scm.CreateBranch) (*scm.Response, error) {
-	return nil, scm.ErrNotSupported
+	namespace, name := scm.Split(repo)
```

Comment:
whats name? repoName or branch name or something else? Can we do better variable naming here.

Reply:
done